### DISCUSSION TITLE
为什么我提交了记录，但在contributions当日没有显示有过提交记录呢？

### DISCUSSION THREAD

**Discussion by wdlami** [community/community#123005] [Repositories]
### Select Topic Area

Product Feedback

### Body

为什么我提交了记录，但在contributions当日没有显示有过提交记录呢？
<img width="919" alt="image" src="https://github.com/community/community/assets/91253689/3fb68cfd-2b49-4b0f-9ee8-c2584cd7eb24">
<img width="1351" alt="image" src="https://github.com/community/community/assets/91253689/4743f8ee-7b2a-4465-b6f1-c1ca49f9d44d">

**Comment by grandmas-favorite** [comment]
Hi @wdlami, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
